.github/workflows/apexE2E.yml

```diff
@@ -93,7 +93,7 @@ jobs:
     with:
       automationBranch: ${{ inputs.automationBranch }}
       testToRun: 'apexLsp.e2e.ts'
-      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      vscodeVersion: ${{ inputs.vscodeVersion }}
```

Comment:
Why isn't `|| 'stable'` needed here?  (And in all the other places where this change was made.)

Reply:
this can only be run from workflow_dispatch or workflow_call, and we provide a value in e2e.yml and one when we manually run the workflow, so it's not needed